source/lac/trilinos_sparse_matrix.cc

```diff
@@ -2215,7 +2215,12 @@ namespace TrilinosWrappers
                                         const_cast<Epetra_CrsMatrix &>(
                                           tmp_result.trilinos_matrix()));
 #  else
-      Assert("false", ExcMessage("This function requires EpetraExt."));
+      Assert("false",
```

Comment:
```suggestion
      Assert("false",
```
Why is this a string?

Reply:
History. Interestingly, this makes the condition `true` :-)